### DISCUSSION TITLE
The Front

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#521] [Game request]
steam: https://store.steampowered.com/app/2285150/The_Front/

servers: https://www.survivalservers.com/wiki/index.php?title=How_to_Create_The_Front_Server_Guide

**Comment by TheHoboArk** [comment]
The Front!

**Comment by scarecr0w12** [comment]
To add some information about the front (Im one of the server owners who've found a lot of info on the admin side for the front).

-There is no Rcon for this, the only technically remote access is via a telnet to the shutdown port over 127.0.0.1 only, which allows graceful shutdowns using the `shutdown` as a command. There are some acct related commands, but nothing else.
-There is no real query for this, however, you can use gamedig to send to the query port acting as Arkse and it will return the Server ID, Max slots, Players in server and number of GMs online. You could then pull more information from the Server list over the fronts api.
-There is no console that is interactable outside of game
-Windows does not always properly run the server, linux is the far better alternative.
-Permissions levels are controlled by the GM.csv file or via adding in game, level s 21-25 are GM levels, 21 being lowest, 25 beign highest.
-Queue skip can be added via PriorityPlayer command in game or adding to GM.csv as a level 20 gm

I've a lot more pinned posts in the server help channel of the fronts discord, but these things should be main points of things to keep note of if adding The front to GSA as a possible server.

**Reply by mvaessen** [reply]
telnet, lol. 

Thanks for the info, very helpful!

**Reply by scarecr0w12** [reply]
Lol yeah i know, it was a weird find, but you can run Shutdown, acct, and a few other things, so im sure its going to act as the rcon port/connection in some fashion going forward.

**Reply by scarecr0w12** [reply]
Oh another note, ALL commands related to a steamid, such as adding users to the GM.csv file, or viewing the character list from the Accounts.csv file (which does show all player characters, their GUID and related steam id for character listing) is setup as `STEAM:steamid`
